plugins/kubernetes/app/models/kubernetes/api/pod.rb

```diff
@@ -0,0 +1,26 @@
+module Kubernetes
+  module Api
+    class Pod
+      def initialize(api_pod)
+        @pod = api_pod
+      end
+
+      def ready?
+        @pod.status.phase == 'Running' && condition_ready?
+      end
+
+      def name
+        @pod.metadata.name
+      end
+
+      private
+
+      def condition_ready?
+        @pod.status.conditions.present? &&
+            @pod.status.conditions
+                .select { |c| c['type'] == 'Ready' }
```

Comment:
Can we expect more that one condition with type 'Ready'? If not, we could use `find` instead of `select`, I think, return that object and then check the status property.